### ISSUE TITLE
Tactics planner: Multiple nobles and fakes

### ISSUE THREAD

**Issue by Butasama** [Torridity/dsworkbench#95]
Hello there!

While using tactics planner, there is an option to send multiple nobles. However, when using exact time for arrival (i.e. 10:00:00.000), nobles from same village are not exported to commands. I suspect that it is because **time of arrival is exactly the same and command tab is incapable of showing two exactly same commands**.

Would it be possible to add 100-200ms offset to noble attacks when using Multiple nobles option? 

My second question is whether it would be possible to allow several fakes from same village to same target because as of now, I have to manually copy paste commands to achieve this result.

Have a great day and thank you for this tool!

**Comment by extremeCrazyCoder** [comment]
>While using tactics planner, there is an option to send multiple nobles. However, when using exact time for arrival (i.e. 10:00:00.000), nobles from same village are not exported to commands. I suspect that it is because time of arrival is exactly the same and command tab is incapable of showing two exactly same commands.

You are right about the saving as multiple commands. If the ms is the same the old command will be overwritten.

> Would it be possible to add 100-200ms offset to noble attacks when using Multiple nobles option?
When reading this I had another option in my mind for fixing this at first. Commands have a field called multiplier that is beeing used to deterimine how often that command should be sent.

But your option with adding 100-200 ms (or better the required spacing on that world) would be more realistic.


> My second question is whether it would be possible to allow several fakes from same village to same target because as of now, I have to manually copy paste commands to achieve this result.

It seems not really that difficult to add thus I will add that

A few questions regarding your observations:
- did you use systematic or random calculation?
- did your villages already perform a non-fake attack on that village?

As it stands now I will most likely add 2 new checkboxes:
- Allow multi attacks of fakes
- Allow faking when there are real attacks

I don't really want to do big changes any more, because I don't really have a way of testing them and I am also working on a new tool will act as an replacement for workbench

**Comment by extremeCrazyCoder** [comment]
A couple of corrections based on the implementation:
- I will not add a checkbox for "Allow faking when there are real attacks" because that will be on by default (implementation is harder than initially thought)

I commited everything now as 770d83fcbda46fd0746cdf8cec6ece140296cb23 

I did not test that at all. If you encounter any bugs please let me know. Since I did not test that I am also not going to create a release immediately
